lib/sharing/ConsumeSharedRuntimeModule.js

```diff
@@ -315,6 +316,7 @@ class ConsumeSharedRuntimeModule extends RuntimeModule {
 									"id",
 									[
 										`if(${RuntimeGlobals.hasOwnProperty}(installedModules, id)) return promises.push(installedModules[id]);`,
```

Comment:
Shouldn't the `installedModules` object already do the same?

Reply:
No, installedModules this is the map of installed modules. startedInstallModules this is the map of started install modules, not loaded and not installed, otherwise "promise looping" in browser runtime.

Reply:
An alternative way to fix it is to remove modules from cache groups. Need add to test function filter by "consume" and "provide" modules.
It seems to me that this is the correct way to fix, if possible remove these modules from user cache groups rules.